src/main.py

```diff
@@ -32,8 +42,10 @@ def signal_handler(sig, frame):
 
 def config_update_callback():
     logger.info("Rebooting the system")
+    logger.debug(f"Sending {RESET_COMMAND} command to {SHUTDOWN_HOST}:{SHUTDOWN_PORT}")
+    comm_module.execute_command(RESET_COMMAND)
```

Comment:
Shouldn't this function be called after the ```stop()```command

Reply:
The stop command stops all the ZeroMQ clients, so if you call stop() before comm_module.execute_command() it will give an error. I could separate the stop function into pieces so the first one stops all except the comm_module and then, after comm_module.execute_command(), stop the comm_module.

Reply:
You are totally right, @aitorcas23, leave it as it is 😉